File: Example Code/01 Variables, Data Types, Input and Output/Math_PricePlusTaxCalculator.cpp
#include <iostream>
using namespace std;

int main()
{
    float price, tax;
    cout << "PRICE PLUS TAX CALCULATOR" << endl;
    
    cout << "Enter price: ";
    cin >> price;
    
    cout << "Enter tax: ";
    cin >> tax;
    
    float pricePlusTax = price + price * tax;
    
    cout << "Price plus tax: " << pricePlusTax << endl;    
    
    return 0;
}

File: Example Code/01 Variables, Data Types, Input and Output/Math_PricePlusTaxCalculator.java
import java.util.Scanner;

public class Math_PricePlusTaxCalculator
{    
    public static void main( String args[] )
    {
        Scanner scanner = new Scanner( System.in );
        
        System.out.println( "PRICE PLUS TAX CALCULATOR" );
        
        System.out.print( "Enter the price: " );
        double price = scanner.nextDouble();
        
        System.out.print( "Enter the tax rate: " );
        double tax = scanner.nextDouble();
        
        double pricePlusTax = price + price * tax;
        
        System.out.println( "Price plus tax: " + pricePlusTax );
    }
}
